draft-wang-ppm-differential-privacy.md

```diff
@@ -491,21 +512,23 @@ This section describes various mechanisms required for implementing DP
 policies. The algorithms are designed to securely expand a short, uniform
 random seed into a sample from a given distribution.
 
-For each mechanism, we expect the noise parameters are computed based on the
-DP guarantee that it is supposed to provide.
+Each mechanism has internal parameters that determine how much noise will be
+added to its input data. The internal parameters are typically computed by
+support methods in order to output the desired `EPSILON`-DP, or
+`(EPSILON, DELTA)`-DP. It is worth noting that a mechanism that is initialized
+with its internal parameters can achieve different combinations of DP
+parameters, e.g. `(EPSILON, DELTA)`-DP, or `(EPSILON', DELTA')`-DP.
```

Comment:
Do we want to explain why this is the case?

Reply:
Explained it a bit:

> where `EPSILON < EPSILON'` and `DELTA > DELTA'`, because if we make `EPSILON` larger (i.e., weaker privacy), we may achieve a smaller `DELTA` (i.e., stronger privacy).